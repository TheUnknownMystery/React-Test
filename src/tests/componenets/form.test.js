import renderer from 'react-test-renderer';
import Form from '../../components/form';
import FormDataFixture from '../fixtures/LinkDataFixture';

test('render form with proper data', () => {
  const form = renderer.create(<Form {...FormDataFixture} />);
  expect(form.toJSON()).toMatchSnapshot();
});

test('should render with default props', () => {
  const form = renderer.create(<Form />);
  expect(form.toJSON()).toMatchSnapshot();
});
