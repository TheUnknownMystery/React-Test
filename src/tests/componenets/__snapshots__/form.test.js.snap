// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render form with proper data 1`] = `
<div>
  <form
    className="form"
  >
    <p>
      Name: 
    </p>
    <input
      name="name"
      onChange={[Function]}
      type="text"
    />
    <p>
      Email: 
    </p>
    <input
      name="email"
      onChange={[Function]}
      type="text"
    />
    <p>
      Message: 
    </p>
    <textarea
      name="message"
      onChange={[Function]}
    />
  </form>
</div>
`;

exports[`should render with default props 1`] = `
<div>
  <form
    className="form"
  >
    <p>
      Name: 
    </p>
    <input
      name="name"
      onChange={[Function]}
      type="text"
    />
    <p>
      Email: 
    </p>
    <input
      name="email"
      onChange={[Function]}
      type="text"
    />
    <p>
      Message: 
    </p>
    <textarea
      name="message"
      onChange={[Function]}
    />
  </form>
</div>
`;
